app/templates/events/trackVolunteers.html

```diff
@@ -76,13 +76,25 @@ <h3>{{eventData["program"].programName}} Event</h3>
       </tr>
     </thead>
     <tbody class="align-middle" id="volunteerTable">
+    {% macro rsvpHistoryData(volunteerUser) %}
+      {% if eventVolunteerData %}
+        {% for volunteer in eventVolunteerData %}
+          {% if volunteerUser == volunteer.user %}
+            <p>{{volunteer.user.firstName}}'s last rsvp was on {{volunteer.rsvpTime.strftime('%b %-d %-I:%M %p')}}</p>
```

Comment:
The rsvp in this sentence should be RSVP

Reply:
fixed!